forge-gui/res/cardsfolder/upcoming/astral_confrontation.txt

```diff
@@ -0,0 +1,7 @@
+Name:Astral Confrontation
+ManaCost:4 W
+Types:Instant
+S:Mode$ ReduceCost | ValidCard$ Card.Self | Type$ Spell | Amount$ X | EffectZone$ All | Description$ CARDNAME costs {1} less to cast for each opponent you're attacking.
+SVar:X:PlayerCountPropertyYou$OpponentsAttackedThisCombat
```

Comment:
Can you add combat = null check here or it will crash if cast in other phases:
https://github.com/Card-Forge/forge/blob/e3c3dd9fc8ba3b06193e5af2bc3711354ff8c807/forge-game/src/main/java/forge/game/ability/AbilityUtils.java#L3489